.bit/responses/5.1-Week 5 Step 1.md

```diff
@@ -0,0 +1,54 @@
+---
+files: n/a
+stepType: checks
+scripts: n/a
+week: 5
+step: 1
+name: Week 5 Step 1
+---
+Week 5 Step 1 ⬤◯◯◯ | 🕐 Estimated completion: 30 minutes
+
+## Laying out the groundwork
+
+### ✅  Task:
+
+- [ ]  Create a new [figma](figma.com) file named "Serverless Camp Project"
+- [ ]  In this file, create a flowchart showing the flow of information and components involved in your project including any Azure Functions and APIs you plan on using
+- [ ]  Export the flowchart as a png file 
+- [ ]  Create a branch named `week5`
+- [ ]  Create a folder in this repo where you will be saving all of your project related files, called "project"
+- [ ]  Upload the flowchart into the project folder and commit your changes to move on
+
+### Mapping it out
+
+To get started with building, students will first need to create a flowchart showing the flow of information and components they will include to make there project work. Nothing has to be finalized, but planning it out allows students to organize their ideas and thought processes.
+
+### Purpose of Programming Flowcharts
+
+- Plan before the development phase helps you save a lot of hours of work
+- Document your project and communicate how it works to others (this is really valuable when someone else is helping you debug your code)
+- Prevent major technical issues with your code before you event start
+- Ensure that you are more effectively using all the technologies available to you
+- You will need to refer back to the flowchart while developing to review your work and sometimes come up with new ideas
+
+<details>
+<summary>:exclamation: How do I create a flowchart?</summary>
+</br>
+Part A: Purpose
+1. Create a checklist of all of the components in your project you want to use, including APIs, functions, databases, packages/libraries, etc.
```

Comment:
this should not be in the dropdown. this information is necessary to complete the assignment